test/Microsoft.DotNet.ProjectModel.Tests/LockFilePatchingTests.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Linq;
+using FluentAssertions;
+using Xunit;
+using Microsoft.DotNet.ProjectModel.Graph;
+using Microsoft.DotNet.Tools.Test.Utilities;
+
+namespace Microsoft.DotNet.ProjectModel.Tests
+{
+    public class LockFilePatchingTests : TestBase
+    {
+
+        private string ExportFilesRoot=> Path.Combine(RepoRoot, "TestAssets", "LockFileSamples", "ExportFileSamples");
+
+        [Fact]
+        public void TestValidPatching()
+        {
+            var lockFilePath = GetLockFilePath("valid");
+            var lockFile = LockFileReader.Read(lockFilePath);
+
+            var exportFile = lockFile.ExportFile;
+
+            exportFile.Should().NotBeNull();
+            exportFile.Exports.Count.Should().Be(3);
+            exportFile.Exports.Should().OnlyHaveUniqueItems();
+
+            // check export structure
+            for (int i = 0; i < 3; i++)
+            {
+                var export = exportFile.Exports.ToList().ElementAt(i);
+
+                export.TargetFramework.Should().NotBeNull();
+
+                AssertTargetLibrary(i + 1, export);
+            }
+
+            lockFile.Targets.Count.Should().Be(3);
+
+            // check lock file structure is similar to export structure
```

Comment:
This seems like an independent test. We're not validating any relationship across these collections, we are independently validating each collection's contents. Can this be split up?